resources.tf

```diff
@@ -1,8 +1,168 @@
+#############  IAM - policy document for breakglass, CI, administrator
+# Create PDM role
```

Comment:
We shouldn't need a new role, and especially not one called "pdm" in the "adg" repo.